pom.xml

```diff
@@ -170,7 +170,7 @@
             <!-- for cernunnos -->
                 <groupId>javax.script</groupId>
                 <artifactId>groovy-engine</artifactId>
-                <version>1.1</version>
+                <version>1.0</version>
```

Comment:
do we need to downgrade the groovy engine?
is it causing an issue with cernunnos?